public/docs/_examples/toh-1/ts/demo.css

```diff
@@ -0,0 +1,9 @@
+body, input[text], button {
```

Comment:
This looks like it's present in the current TOH as well, but is the `input[text]` selector doing anything? AFAIK this will target an input element with an attribute named 'text'. I'm guessing `input[type="text"]` was the intent?